src/easy_config/__init__.py

```diff
@@ -148,12 +149,14 @@ def load(
 
         1. values from the files listed in the FILES class variable, in order
         2. values from files passed in the additional_files parameter, in order
-        3. values from the environment
-        4. values passed as keyword arguments to this method (useful for values specified on the command line)
+        3. values from the file specified by the config file specified by the environment variable _lookup_config_envvar
```

Comment:
"values from the file specified by the path in the environment variable _lookup_config_envvar" reads better.

Also it doesn't specify the NAME_ and upper transformation here.